ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -406,7 +406,8 @@ public void setAllowFontScaling(ReactEditText view, boolean allowFontScaling) {
 
   @ReactProp(name = "placeholder")
   public void setPlaceholder(ReactEditText view, @Nullable String placeholder) {
-    view.setHint(placeholder);
+    String text = placeholder == null ? "" : placeholder;
```

Comment:
the issue here is `@Nullable String placeholder` so I will try remove `@Nullable` option